python/doc/theory/numerical_methods/rosenblatt_transformation.rst

```diff
@@ -3,51 +3,54 @@
 Rosenblatt Transformation
 -------------------------
 
-| The Rosenblatt transformation is an isoprobabilistic transformation
-  (refer to ) which is used under the following context :
-  :math:`\vect{X}` is the input random vector, :math:`F_i` the
-  cumulative density functions of its components and :math:`C` its
-  copula, without no condition on its type.
-| Let us denote by :math:`\vect{d}` a deterministic vector,
-  :math:`g(\vect{X}\,,\,\vect{d})` the limit state function of the
-  model,
-  :math:`\cD_f = \{\vect{X} \in \Rset^n \, / \, g(\vect{X}\,,\,\vect{d}) \le 0\}`
-  the event considered here and g(,) = 0 its boundary.
-| One way to evaluate the probability content of the event
-  :math:`\cD_f`:
-
-  .. math::
+The Rosenblatt transformation is an :ref:`isoprobabilistic transformation <isoprobabilistic_transformation>` which is used under
```

Comment:
This comment is on the wrong place with respect to the location of the bug, but I cannot reach the end of the file because you did not modify it.

The "References" section has an itemized list which is poorly formatted:

![image](https://user-images.githubusercontent.com/31351465/235197393-a950d8b0-6a15-4cda-97ec-0c9106019a0d.png)

Please apply the same bug fix as before, cutting the long line into several pieces.

Reply:
ok